lib/code-caller/code-caller-native.js

```diff
@@ -212,8 +215,8 @@ class CodeCallerNative {
 
     this.lastCallData = callData;
 
-    this.child.stdin.write(callDataString);
-    this.child.stdin.write('\n');
+    this.child?.stdin?.write(callDataString);
+    this.child?.stdin?.write('\n');
```

Comment:
Would it be better to assert that `this.child` is non-null here? `this._checkState([WAITING])` above does this check, but not in a way that TypeScript is aware of 😢

Reply:
I've used assertions for this sort of case and I find them a reasonable pattern when it substantially simplifies the following code. I'm not sure in this case that it will really make much difference either way, so I'm agnostic.